example/webpack.config.js

```diff
@@ -16,7 +16,7 @@ module.exports = {
   resolve: {
     alias: {
       // Use uncompiled version
-      //'react-schema-form': '../src'
+      'react-schema-form': '../src',
```

Comment:
This is useful for testing, but I think it is better to use the compiled version of the library for the examples page

Reply:
sure, will change